understanding/content-on-hover-or-focus.html

```diff
@@ -55,348 +53,279 @@
+                  <p>この条件の意図は、利用者に追加コンテンツが可視になった後、知覚するための十分な時間があるようにすることである。障害のある利用者は拡大率を変更する、ポインタを動かす、又は可視領域に新しいコンテンツを持ってくるなどの様々な理由により、より多くの時間が必要かもしれない。一度現れたら、次のことをする時までコンテンツは可視のままにするべきである:
+
+
                   </p>
                   			
                   <ul>
                      			
-                     <li>The user removes hover or focus from the trigger and the additional content, consistent
-                        with the typical user experience;
+                     <li>利用者が、典型的なユーザ体験と一致して、トリガー及び追加コンテンツからホバー又はフォーカスを取り除く。
                      </li>
                      			
-                     <li>The user dismisses the additional content via the mechanism provided to satisfy the
-                        Dismissable condition; or
+                     <li>利用者が非表示にすることができる条件を満たすために提供されたメカニズムを通して追加コンテンツを非表示にする。又は、
                      </li>
                      			
-                     <li>The information conveyed by the additional content becomes invalid, such as a 'busy'
-                        message that is no longer valid.
+                     <li>もはや有効ではない「使用中」メッセージなど、追加コンテンツにより伝えられた情報が無効になる。
                      </li>
                      			
                   </ul>
                   			
                </section>
                <section id="additional-notes">
                   			
-                  <h3>Additional Notes</h3>
+                  <h3>追加の注記</h3>
                   			
                   <ul>
                      			
-                     <li>This criterion does not attempt to solve such issues when the appearance of the additional
-                        content is completely controlled by the user agent. A prominent example is the common
-                        behavior of browsers to display the <code>title</code>  attribute in HTML as a small tooltip.
+                     <li>この基準は、追加コンテンツの見た目が完全にユーザエージェントにより制御されている時に、このような問題を解決することに努めていない。顕著な例は、HTML の <code>title</code> 属性を小さなツールチップとして表示するブラウザの一般的な挙動である。
                      </li>
                      			
-                     <li>Modal dialogs are out of scope for this criterion because they must take keyboard
-                        focus  and thus should not appear on hover or focus.  Refer to <a href="https://www.w3.org/TR/WCAG21/#on-focus">Success Criterion 3.2.1, On Focus</a>.
+                     <li>モーダルダイアログはキーボードフォーカスを受け取らなければならないため、ホバー又はフォーカスで現れるべきではなく、この基準のスコープ外である。 <a href="https://waic.jp/translations/WCAG22/#on-focus">達成基準 3.2.1 フォーカス時</a>を参照すること。
                      </li>
                      			
-                     <li>Content which can be triggered via pointer hover should also be able to be triggered
-                        by keyboard focus.  Refer to <a href="https://www.w3.org/TR/WCAG21/#keyboard">Success Criterion 2.1.1, Keyboard</a>.
+                     <li>ポインタホバーを通して動作できるコンテンツはキーボードフォーカスでも動作できるべきである。<a href="https://waic.jp/translations/WCAG22/#keyboard">達成基準 2.1.1 キーボード</a>を参照すること。
```

Comment:
「動作できる」は変かも? 「動作する」というだけでも can be のニュアンスはある程度カバーできような気がします。

```suggestion
                     <li>ポインタホバーを通して動作するコンテンツはキーボードフォーカスでも動作するべきである。<a href="https://waic.jp/translations/WCAG22/#keyboard">達成基準 2.1.1 キーボード</a>を参照すること。
```